src/index.js

```diff
@@ -223,6 +223,8 @@ class View extends React.Component {
     } = this.state
     const { createRoom, createConvo, removeUserFromRoom } = this.actions
 
+    console.log(messages);
```

Comment:
This probably doesn't want to make it on to master.

Reply:
oops missed that one, good pick up. Its removed now.

Reply:
@mdpye can you re-review please?